broker/test/test_broker/utils.spec.js

```diff
@@ -505,15 +505,15 @@ describe('utils', function () {
           }
         }]
       };
-      mocks.apiServerEventMesh.nockGetResourcesAcrossAllNamespaces(CONST.APISERVER.RESOURCE_GROUPS.INTEROPERATOR, CONST.APISERVER.RESOURCE_TYPES.INTEROPERATOR_SERVICES, expectedResponse);
+      mocks.apiServerEventMesh.nockGetResourcesAcrossAllNamespaces(CONST.APISERVER.RESOURCE_GROUPS.INTEROPERATOR, CONST.APISERVER.RESOURCE_TYPES.INTEROPERATOR_SERVICES, expectedResponse, {});
       return utils.getAllServices()
         .then(res => {
           expect(res).to.eql([expectedResponse.items[0].spec]);
           mocks.verify();
         });
     });
     it('Throws error on getting list of services from apiserver', () => {
-      mocks.apiServerEventMesh.nockGetResourcesAcrossAllNamespaces(CONST.APISERVER.RESOURCE_GROUPS.INTEROPERATOR, CONST.APISERVER.RESOURCE_TYPES.INTEROPERATOR_SERVICES, {}, undefined, 1, 500);
+      mocks.apiServerEventMesh.nockGetResourcesAcrossAllNamespaces(CONST.APISERVER.RESOURCE_GROUPS.INTEROPERATOR, CONST.APISERVER.RESOURCE_TYPES.INTEROPERATOR_SERVICES, {}, {}, 1, 500);
```

Comment:
`mocks.apiServerEventMesh.nockGetResourcesAcrossAllNamespaces` takes a query string as its 4th argument. From nock v11.0.0 query string cannot be `undefined`. Passing an empty `{}` instead.